src/olympia/lib/settings_base.py

```diff
@@ -1759,10 +1752,22 @@ def read_only_mode(env, retry_after=None):
     path('static'),
 )
 
-NETAPP_STORAGE = TMP_PATH
-GUARDED_ADDONS_PATH = os.path.join(ROOT, 'guarded-addons')
+# Path related settings. In dev/stage/prod `NETAPP_STORAGE_ROOT` environment
+# variable will be set and point to our NFS/EFS storage
+# Make sure to check overwrites in conftest.py if new settings are added
+# or changed.
+STORAGE_ROOT = env('NETAPP_STORAGE_ROOT', default=path('storage'))
+
+ADDONS_PATH = os.path.join(STORAGE_ROOT, 'files')
```

Comment:
Note: this is actually a change - `ADDONS_PATH` was undefined in `settings_base.py` so using `user_media_path` would default to doing `os.path.join(settings.MEDIA_ROOT, 'addons')` instead (rightly or wrongly).

Reply:
Correct, only for local development though since it's been using `join(STORAGE_ROOT, 'files') on -dev, -stage and -prod. I hoped that'd be an okay compromise

Reply:
yeah we'll just have to remember to move any files we have in our local dev after this is in master

Reply:
I'll send an email to addons-dev before I merge this.